src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Api.Enduser/Models/ConnectionExternal.cs

```diff
@@ -0,0 +1,6 @@
+namespace Altinn.AccessManagement.Api.Enduser.Models;
+
+public class ConnectionExternal
```

Comment:
Vi prøver vel å gå bort fra External navngivning og i stedet bruke Dto for "eksterne" modeller

Reply:
slettet modell